tests/notehead_tests.ts

```diff
@@ -72,15 +67,15 @@ function basic(options: TestOptions, contextBuilder: ContextBuilder): void {
 /**
  * Used by the next two test cases to draw a note.
  */
-function showNote(struct: StaveNoteStruct, stave: Stave, ctx: RenderContext, x: number) {
-  const note = new StaveNote(struct).setStave(stave);
+function showNote(noteStruct: Partial<StaveNoteStruct>, stave: Stave, ctx: RenderContext, x: number) {
+  const note = new StaveNote(noteStruct).setStave(stave);
   new TickContext().addTickable(note).preFormat().setX(x);
   note.setContext(ctx).draw();
   return note;
 }
 
 function variousHeads(options: TestOptions, contextBuilder: ContextBuilder): void {
-  const notes: StaveNoteStruct[] = [
+  const notes: Partial<StaveNoteStruct>[] = [
```

Comment:
Hmm... I'm not sure I'm a fan of seeing the `Partial<...>` leak into the tests files. That's a sign that the developer who runs `npm install vexflow` will have to utilize `Partial<...>` in his/her code, simply because we choose to accept Partials.

Reply:
I do not quite agree. We have to use Partial due to the fact that we decided to use Partial definitions in the tests. If we defined the complete structure in the test Partial would not be required.